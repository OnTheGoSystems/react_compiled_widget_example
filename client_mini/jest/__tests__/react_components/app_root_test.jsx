'use strict';

import React from 'react';
import AppRoot from 'root/components/app_root';
import renderer from 'react-test-renderer';

describe('AppRoot', ()=>{
  it('#render', () => {
    const component = renderer.create(<AppRoot/>);
    const json = component.toJSON();
    expect(json).toMatchSnapshot();
  });
});