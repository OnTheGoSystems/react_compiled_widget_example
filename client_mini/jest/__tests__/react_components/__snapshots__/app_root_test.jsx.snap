// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppRoot #render 1`] = `
<div
  className="app_root_container"
>
  <h1>
    Compiled react widget
  </h1>
  <div>
    <h3>
      Notification
    </h3>
    <p>
      <button
        onClick={[Function]}
      >
         To Dashboard 
      </button>
    </p>
    <p>
      example
    </p>
  </div>
</div>
`;
